script.js

```diff
@@ -1,12 +1,265 @@
-//You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+async function setup() {
+  let allShows = getAllShows();
+  let urlForApi = `${allShows[0]._links.self.href}/episodes`;
+  let allEpisodes = await fetchFromApi1(urlForApi); //this line might be usfule when passing url to make it general ...let urlOfApi = `${shows[event.target.value]._links.self.href}/episodes`;
+  makePageForShows(allShows, allEpisodes);
+  // selectShow(allShows);
+  // selectEpisode(allEpisodes);
+  // makePageForEpisodes(allEpisodes);
+  // makeLiveSearch(allEpisodes);
+}
+
+function makePageForShows(shows, episodes) {
+  /*When your app starts, present a listing of all shows ("shows listing")
+For each show, you must display at least
+name, image, summary, genres, status, rating, and runtime.*/
+
+  const mainContentElem = document.getElementById("content");
+
+  //clear everything before we start
+  mainContentElem.innerHTML = " ";
+
+  for (let i = 0; i < shows.length; i++) {
```

Comment:
You could also make use of the built in method `forEach` here, for example `shows.forEach`